tfw

```diff
@@ -151,6 +165,23 @@ Administrivia:
   admin-users
 
 EOUSAGE
+
+  shopt -s nullglob
+  local external_commands=()
+  for path_prefix in ${PATH//:/ }; do
+    for exe in "${path_prefix}"/tfw-*; do
+      external_commands=("${external_commands[@]}" "${exe##*tfw-}")
```

Comment:
What does `"${exe##*tfw-}")` do?

Reply:
Strip `*tfw-` from the front of the string, equivalent to `string.gsub(/^.*tfw-/, '')`